src/handwritten/system/_collections/device_collection.py

```diff
@@ -15,6 +15,9 @@ class DeviceCollection(Sequence):
     
     This class defines methods that implements a container object.
     """
+    def __init__(self, interpreter):
+        self._interpreter = interpreter
```

Comment:
@zhindes Are these collections public APIs? They live in private sub-modules, but they are returned from public classes.

Should we continue to allow constructing a `DeviceCollection()`?

Reply:
I'm fine if these break compatibility as long as the public classes usage is maintained.

Reply:
@MounikaBattu17 Let's make sure the documentation is clear about this:
- Add a docstring to the collection classes' `__init__` methods saying do not construct this object directly; instead, call `nidaqmx.system.System.local().devices`, etc.
- Add a note to CHANGELOG.md saying that the signature of the collection classes' `__init__` methods have changed and any code that constructs these objects directly should be updated to get them from the appropriate object (e.g. `System.devices`, `Task.channels`).

Reply:
Thanks for the detailed comment. Updated the docstrings for all collection classes and added the change in `CHANGELOG.md` in a new version `0.7.2`. Please specify if this version is correct or not.